test/access/AccessControl.behavior.js

```diff
@@ -215,13 +215,21 @@ function shouldBehaveLikeAccessControlEnumerable(errorPrefix, admin, authorized,
 function shouldBehaveLikeAccessControlDefaultAdminRules(errorPrefix, delay, defaultAdmin, newDefaultAdmin, other) {
   shouldSupportInterfaces(['AccessControlDefaultAdminRules']);
 
-  it('has a default disabled delayed until', async function () {
-    expect(await this.accessControl.defaultAdminTransferDelayedUntil()).to.be.bignumber.equal(web3.utils.toBN(0));
-  });
+  // it('has an admin transfer schedule disabled by default', async function () {
```

Comment:
I think this is covered in your to-do list, but so we don't forget I'll say that these tests are commented out.

Reply:
Yes, updated. I'm solving since there are replacing tests.